commons/zenoh-config/src/lib.rs

```diff
@@ -70,6 +70,18 @@ impl Zeroize for SecretString {
 
 pub type SecretValue = Secret<SecretString>;
 
+#[derive(Debug, Deserialize, Serialize, Clone)]
```

Comment:
Do we have some tool for generate DEFAULT_CONFIG.json5 file or should sync/modify it manually?

Reply:
The sync is done manually for the time bein.